### DISCUSSION TITLE
Adding a new column to a table in the DB, how to go about it?

### DISCUSSION THREAD

**Discussion by mdemoss** [FreshRSS/FreshRSS#3873] [Q&A]
I'm re-working the shuffle-sort feature I wrote a bit ago, and I want to add a `whenRead` column to the entry table.

What's the best way to go about that so the schema gets upgraded nicely and nothing breaks?

**Comment by mdemoss** [comment]
**checkInstallAction()** calls **check_install_database()** calls **entryIsCorrect()** calls **checkTable()**

I can't see anywhere checkInstallAction() is called.

**Reply by Frenzie** [reply]
Search for `checkInstall`, not `checkInstallAction`. It should be run when you click "Update" after checking for updates.
